packages/happy-dom/src/nodes/document/Document.ts

```diff
@@ -1165,8 +1165,13 @@ export default class Document extends Node {
 	 * @param [data] Text data.
 	 * @returns Text node.
 	 */
-	public createTextNode(data?: string): Text {
-		return NodeFactory.createNode<Text>(this, this[PropertySymbol.ownerWindow].Text, data);
+	public createTextNode(data: string): Text {
```

Comment:
In TypeScript `lib.dom.d.ts`, the createTextNode argument is not optional.

<img width="628" alt=" 2024-04-06 1 57 31" src="https://github.com/capricorn86/happy-dom/assets/6040962/322c022b-a367-4916-bb2f-cad31525a0dd">


I have changed the type and implementation to mimic browser behavior.

| Safari | Firefox | Chrome |
|-|-|-|
|<img width="251" alt=" 2024-04-06 1 50 46" src="https://github.com/capricorn86/happy-dom/assets/6040962/a113318e-e926-44c9-8a47-f18ede14e9d3">|<img width="649" alt=" 2024-04-06 1 51 00" src="https://github.com/capricorn86/happy-dom/assets/6040962/97f1f91e-40e4-4ce8-8a35-29d8cda116ad">|<img width="562" alt=" 2024-04-06 1 50 53" src="https://github.com/capricorn86/happy-dom/assets/6040962/d4a796e4-7e7f-452e-811f-28b8bab62f44">|

Could you help me determine whether this change is considered a 'bug fix' or a 'breaking change'?